calico/observability/view-flow-logs.mdx

```diff
@@ -3,4 +3,98 @@ description: View flow logs in the Calico Whisker web console.
+| `packets_out`                     | long              | Number of outgoing packets since the last export.                                                                                                                                                                                                                                                                                                    |
+| `protocol`                        | keyword           | Protocol.                                                                                                                                                                                                                                                                                                                                            |
+| `policies`                        | array of arrays   | List of policies that interacted with this flow. See [Format of the policies field](#format-of-the-policies-field).                                                                                                                                                                                                                                                |
+| `source_name`                     | keyword           | Contains one of the following values: <br />- Name of the source pod.<br />- Name of the pod that was aggregated or the endpoint is not a pod. Check <code>source_name_aggr</code> for more information, such as the name of the pod if it was aggregated.                                                                                           |
+| `source_namespace`                | keyword           | Namespace of the source endpoint. A `-` means the endpoint is not namespaced.                                                                                                                                                                                                                                                                        |
+| `source_labels`                   | array of keywords | Labels applied to the source pod. A hyphen indicates aggregation.                                                                                                                                                                                                                                                                                    |
+
+## View flow logs in the web console
+
+You can quickly view flow logs by port forwarding the Calico Whisker web console to your workstation.
+For security reasons, Whisker does not automatically expose an endpoint.
```

Comment:
I think it's fine to say "By default" instead of "For security reasons" - it makes it sound like there is something intrinsically insecure about Whisker, which isn't true - just that we can't guess the right way to export this Service outside of the cluster, and operationalizing a UI with authentication is typically a bespoke process per-user.

Reply:
Changed.